DEVELOPING.md

```diff
@@ -24,7 +24,7 @@ First upgrade pip and then install the dependencies
 Get a client ID and secret, if you don't already have one.
 
 ```
-(.venv) $ curl --request POST --header "Content-Type: application/json" --data '{"redirect_uris": ["http://localhost:8000/oidc/callback/"], "application_type": "native", "token_endpoint_auth_method": "client_secret_post"}' https://iddev.fedorainfracloud.org/openidc/Registration
+(.venv) $ curl --request POST --header "Content-Type: application/json" --data '{"redirect_uris": ["http://localhost:1234", "http://localhost:8000/oidc/callback/"], "application_type": "native", "token_endpoint_auth_method": "client_secret_post"}' https://iddev.fedorainfracloud.org/openidc/Registration
```

Comment:
What is this second `redirect_uri` for?

Reply:
It is currently needed by the library openidc-client, so here it is really an hack to make it work.